### PR TITLE
attempt to use github ci instead of travis

### PR DESCRIPTION
**Pull request by sriraamas**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+      - master
+    tags:
+      - "*.*.*"
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  build-and-test:
```
Comment (tmccombs) [inline .github/workflows/ci.yml]: this doesn't actually run any tests...

**Review by tmccombs — APPROVED** [review]

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+        with:
+          path: |
+            ~/.ivy2
+            ~/.sbt
+      - name: Set SBT_OPTS
+        run: bash gen_sbt_opts.sh
+      - name: Publish to Sonatype
+        run: bash publish.sh
+        env:
+          PGP_SECRET: ${{ secrets.PGP_SECRET }}
```
Comment (tmccombs) [inline .github/workflows/ci.yml:47]: have these secrets been added to the repo?

Comment (sriraamas) [reply .github/workflows/ci.yml:47]: no. they have not been added. but I wasn't sure how to get them. is this setup in travis somewhere?

File: .github/workflows/ci.yml (hunk 1 above)
Comment (sriraamas) [reply .github/workflows/ci.yml]: true. will rename

**Pull request merged**